deny.toml

```diff
@@ -0,0 +1,9 @@
+[licenses]
+allow = ["Apache-2.0", "CC-BY-3.0", "ISC", "AFL-2.1", "AFL-3.0", "Artistic-1.0",
+"Artistic-2.0", "Apache-1.1", "BSD-1-Clause", "BSD-2-Clause", "BSD-3-Clause",
+"0BSD", "FTL", "LPL-1.02", "MS-PL", "MIT", "NCSA", "OpenSSL", "PHP-3.0", "TCP-wrappers", 
+"W3C", "Xnet", "Zlib", "CC0-1.0", "APSL-2.0", "CDDL-1.0", "CDDL-1.1", "CPL-1.0", 
+"EPL-1.0", "IPL-1.0", "MPL-1.0", "MPL-1.1", "MPL-2.0", "MPL-2.0-no-copyleft-exception",
```

Comment:
Sorry wasn't clear: the `reciprocal_licenses` are also banned. Not sure why we have them split out from `restricted_licenses`...

The being said, none of them are used so we are in the clear.

Also, nitpick:  it might be nice to have these 1-per-line